zebra-test/src/command.rs

```diff
@@ -241,6 +241,37 @@ impl TestOutput {
         .with_section(stdout)
     }
 
+    pub fn stdout_equals(&self, s: &str) -> Result<&Self> {
+        let stdout = String::from_utf8_lossy(self.output.stdout.as_slice());
```

Comment:
Fun Fact, incase you're unfamiliar, but this can be done without the `.as_slice()` by using "Deref Coercion". https://doc.rust-lang.org/nightly/nomicon/coercions.html

```suggestion
        let stdout = String::from_utf8_lossy(&self.output.stdout);
```